src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs

```diff
@@ -546,137 +543,197 @@ public enum WebRTCStatisticsProperty
+        PacketsLost,
 
         /// <summary>
         /// Discarted packet.
         /// </summary>
-        PacketsDiscarded = WebRTCStatisticsPropertyCategory.ReceivedRtpStream | 0x03,
+        PacketsDiscarded,
 
         /// <summary>
         /// Jitter.
         /// </summary>
-        Jitter = WebRTCStatisticsPropertyCategory.ReceivedRtpStream | 0x05,
+        Jitter,
 
         /// <summary>
         /// Received bytes.
         /// </summary>
-        BytesReceived = WebRTCStatisticsPropertyCategory.InboundRtpStream | 0x01,
+        BytesReceived,
 
         /// <summary>
         /// Duplicated packet.
         /// </summary>
-        PacketsDuplicated = WebRTCStatisticsPropertyCategory.InboundRtpStream | 0x02,
+        PacketsDuplicated,
 
         /// <summary>
         /// Sent bytes.
         /// </summary>
-        BytesSent = WebRTCStatisticsPropertyCategory.SentRtpStream | 0x01,
+        BytesSent,
 
         /// <summary>
         /// Sent packets.
         /// </summary>
-        PacketsSent = WebRTCStatisticsPropertyCategory.SentRtpStream | 0x02,
+        PacketsSent,
 
         /// <summary>
         /// Remote ID.
         /// </summary>
-        RemoteId = WebRTCStatisticsPropertyCategory.InboundRtpStream | WebRTCStatisticsPropertyCategory.OutboundRtpStream | 0x01,
+        RemoteId,
 
         /// <summary>
         /// FIR count.
         /// </summary>
-        FirCount = WebRTCStatisticsPropertyCategory.InboundRtpStream | WebRTCStatisticsPropertyCategory.OutboundRtpStream | 0x02,
+        FirCount,
 
         /// <summary>
         /// PLI count.
         /// </summary>
-        PliCount = WebRTCStatisticsPropertyCategory.InboundRtpStream | WebRTCStatisticsPropertyCategory.OutboundRtpStream | 0x03,
+        PliCount,
 
         /// <summary>
         /// NACK count.
         /// </summary>
-        NackCount = WebRTCStatisticsPropertyCategory.InboundRtpStream | WebRTCStatisticsPropertyCategory.OutboundRtpStream | 0x04,
+        NackCount,
 
         /// <summary>
         /// Round trip time.
         /// </summary>
-        RoundTripTime = WebRTCStatisticsPropertyCategory.RemoteInboundRtpStream | 0x01,
+        RoundTripTime,
 
         /// <summary>
         /// Lost fraction.
         /// </summary>
-        FractionLost = WebRTCStatisticsPropertyCategory.RemoteInboundRtpStream | 0x02,
+        FractionLost,
 
         /// <summary>
         /// Remote timestamp.
         /// </summary>
-        RemoteTimestamp = WebRTCStatisticsPropertyCategory.OutboundRtpStream | 0x01,
+        RemoteTimestamp,
+
+        /// <summary>
+        /// Local ID.
+        /// </summary>
+        LocalId,
+
+        /// <summary>
+        /// Kind.
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        Kind,
+
+        /// <summary>
+        /// Address of the candidate.
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        Address,
 
         /// <summary>
         /// Local ID.
```

Comment:
that summary is attached to the Port property, the Local ID is at line 674

Reply:
I'll fix it.